test/plugins/builds.test.js

```diff
@@ -3182,7 +3257,7 @@ describe('build plugin test', () => {
                     // If user restarts `a`, it should get `d`'s parent event status and trigger `c`
                     const buildC = {
                         jobId: 4,
-                        status: 'SUCCESS',
+                        status: 'CREATED',
```

Comment:
I think it's wrong that the status is success because it's used for updatedBuildC, so I'll fix it.